### ISSUE TITLE
field lookup

### ISSUE THREAD

**Issue by Ian-Mint** [SubratoChakravorty/ECE-229-Group5-final-project#23]
Right now, we populate the dropdowns with the short text, but the user has no way to lookup long text.

**Comment by Ian-Mint** [comment]
dropdowns don't natively support hover

**Comment by wangjinjie722** [comment]
Still cannot find a way to add hover to dropdown...

**Comment by Ian-Mint** [comment]
Yeah, I think we just need to make a reference table at the bottom. I also
might make the plot hover display the long description.

**Comment by Ian-Mint** [comment]
Added a reference table at the end of the dashboard
